osf_tests/test_institution.py

```diff
@@ -55,3 +55,29 @@ def test_institution_banner_path():
     inst = InstitutionFactory(banner_name='osf-banner.png')
     expected_banner_path = '/static/img/institutions/banners/osf-banner.png'
     assert inst.banner_path == expected_banner_path
+
+
+class TestInstitutionPermissions:
+
+    @pytest.fixture()
+    def institution(self):
+        return InstitutionFactory()
+
+    @pytest.fixture()
+    def user(self):
+        return AuthUserFactory()
+
+    @pytest.fixture()
+    def institution_admin_user(self, institution, user):
```

Comment:
if these tests are expanded in the future (or used as a pattern for other metrics tests) and someone tries to use both `user` and `institution_admin_user` fixtures in the same test, it'll be a problem -- they are the same user.

better to make a separate user in this fixture
```suggestion
    def institution_admin_user(self, institution):
```

Reply:
Good suggestion!